configure.scss

```diff
@@ -0,0 +1,53 @@
+$_n-ui-features: (
+	ads: false,
+	buttons: false,
+	grid: false,
+	colors: false,
+	icons: false,
+	typography: false,
+	cookieMessage: false,
```

Comment:
Oh yeah, and everything in Sass is lower case hyphenated - the only time I've seen camelCase in Sass is Origami. I wonder if we should pick a side.